Sources/Profile-View.php

```diff
@@ -1075,8 +1075,8 @@ function list_getNumAttachments($boardsAllowed, $memID)
 		SELECT COUNT(*)
 		FROM {db_prefix}attachments AS a
 			INNER JOIN {db_prefix}messages AS m ON (m.id_msg = a.id_msg)
-		WHERE {query_see_message_board}
-			AND a.attachment_type = {int:attachment_type}
+			INNER JOIN {db_prefix}boards AS b ON (b.id_board = m.id_board AND {query_see_board})
```

Comment:
I don't know why you reverted the change that removed this JOIN. I think it would have been better to use `{query_see_message_board}` and just change `b.id_board` to `m.id_board` in the following lines.

Reply:
I see not much chance that this change the runtime behavior of this query, 
So keep it how it was should be in general not wrong.

Reply:
Fair enough.